src/schema/index.ts

```diff
@@ -182,20 +183,27 @@ const Schema = gql`
     description: String
   }
 
-  type Rating {
-    id: ID!
+    type Rating {
     user: User!
     sprint: Int!
     phase: String!
     quantity: String!
+    quantityRemark: String
+    bodyQuantity: String
     quality: String!
+    qualityRemark: String
     attendance: String
+    bodyQuality: String
     professional_Skills: String!
+    professionalRemark: String
+    bodyProfessional: String
```

Comment:
We have removed some fields like quantityRemark and qualityRemark. It seems you are re-introducing them.

Reply:
My bad , I have fixed this issue. Thanks for  pointing it out.